File: 01.Java-Basic/06.ConditionalStatementsAdvancedExercise/src/SummerOutfit.java
import java.util.Scanner;

public class SummerOutfit {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int degrees = Integer.parseInt(scanner.nextLine());
        String time = scanner.nextLine();
        String outfit = "";
        if (degrees >= 10 && degrees <= 18) {
            if (time.equals("Morning")) {
                outfit = "Sweatshirt and Sneakers";
            } else if (time.equals("Afternoon") || time.equals("Evening")) {
                outfit = "Shirt and Moccasins";
            }
        } else if (degrees > 18 && degrees <= 24) {
            if (time.equals("Morning") || time.equals("Evening")) {
                outfit = "Shirt and Moccasins";
            } else if (time.equals("Afternoon")) {
                outfit = "T-Shirt and Sandals";
            }
        } else if (degrees >= 25) {
            if (time.equals("Morning")) {
                outfit = "T-Shirt and Sandals";
            } else if (time.equals("Afternoon")) {
                outfit = "Swim Suit and Barefoot";
            } else if (time.equals("Evening")) {
                outfit = "Shirt and Moccasins";
            }
        }

        System.out.printf("It's %d degrees, get your %s.", degrees, outfit);
    }
}

File: JavaScript/01.JavaScriptBasic/06Nested-Conditional-Statements-Еxercise/src/SummerOutfit.js
function chooseOutfit(params) {
    let degrees = Number(params[0]);
    let period = params[1];

    let outfit = "";

    switch (period) {
        case "Morning":
            if (degrees >= 10 && degrees <= 18) {
                outfit = "Sweatshirt and Sneakers";
            }  else if (degrees > 18 && degrees <= 24) {
                outfit = "Shirt and Moccasins";
            } else {
                outfit = "T-Shirt and Sandals";
            }
            break;

        case "Afternoon":
            if (degrees >= 10 && degrees <= 18) {
                outfit = "Shirt and Moccasins";
            } else if (degrees > 18 && degrees <= 24) {
                outfit = "T-Shirt and Sandals";
            } else {
                outfit = "Swim Suit and Barefoot";
            }
            break;

        case "Evening":
            outfit = "Shirt and Moccasins";
            break;

        default:
            break;
    }

    console.log(`It's ${degrees} degrees, get your ${outfit}.`);
}

chooseOutfit(["22", "Afternoon"])